Sources/Subs-Db-mysql.php

```diff
@@ -60,6 +60,7 @@ function smf_db_initiate($db_server, $db_name, $db_user, $db_passwd, $db_prefix,
 			'db_mb4'                    => false,
 			'db_ping'                   => 'mysqli_ping',
 			'db_fetch_all'              => 'smf_db_fetch_all',
+			'db_error_insert' => 'smf_db_error_insert',
```

Comment:
Can you keep the indentation?

Reply:
when you look across the different db file than you notice that the laste line is the correct one.
But i align the subs db one now.